chirpstack-concentratord-sx1301/src/config/vendor/rakwireless/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod rak2245_pi_hat_as923;
```

Comment:
Do you know if there is a difference between the rak2245 Pi Hat and other 2245 models? As far as I know it is exactly the same module, but soldered on a PCB that fits on a Pi. If there is no difference, then maybe there is no need to add the `_pi_hat` in the model name :)

Reply:
I know that there are several hardware gateways that RAK offers with this model but I am not aware if all their products that use RAK2245 are the same in terms of pcb components, in any case, I would rather move taking away _pi_hat and if somebody uses another RAK2245 with some difference, then updated it as we move along.

Reply:
Sounds good to me :+1: